src/pages/user/components/GameBoard.jsx

```diff
@@ -1,14 +1,23 @@
-import { useState } from "react";
+import { useState, useEffect } from "react";
 
 const initialBoard = [
   [null, "red", null, "red"],
   [null, null, null, null],
   [null, null, null, null],
   ["black", null, "black", null],
 ];
-export default function GameBoard() {
+export default function GameBoard({ players }) {
   const [board, setBoard] = useState(initialBoard);
   const [selectedPiece, setSelectedPiece] = useState(null);
+  const [gameStarted, setGameStarted] = useState(false);
+
+  useEffect(() => {
+    if (players.length === 2) {
+      setGameStarted(true);
+    } else {
+      setGameStarted(false);
+    }
```

Comment:
How do you feel about if/else? What if I were to show you...

`setGameStarted(players.length === 2);`

Reply:
That's brilliant! It looks more clean than if/else statement here. Thank you!